src/components/Link/Link.js

```diff
@@ -71,6 +71,14 @@ const Link = ({
     );
   }
 
+  if (forceOpenInSameTab) {
```

Comment:
The previous implementation was causing a warning in the console because `<GatsbyLink />` should never be used for external URLs. We always want internal URLs to open in the same tab anyway, so this implementation is more correct.